lib/ord.js

```diff
@@ -165,6 +172,7 @@ module.exports = (csn) => {
     const eventResources = _getEventResources(linkedCsn, appConfig, packageIds);
     if (eventResources.length) {
         ordDocument.eventResources = eventResources;
+        ordDocument.integrationDependency = _getIntegrationDependencies(linkedCsn, appConfig, packageIds);
```

Comment:
this is not the right approach to build the integration dependencies, you have to read the package json/cds config for this, the information is not the same as the events